src/devices/Vcnl4040/samples/Explorer/.gitignore

```diff
@@ -0,0 +1 @@
+output
```

Comment:
Why is this required? Test output should be below the `bin` folder, not in the source folder.

Reply:
This is an artifact of my own deployment script. I'll remove it.